x-pack/plugins/security_solution/public/cases/components/all_cases/status_filter.test.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { mount } from 'enzyme';
+import { waitFor } from '@testing-library/react';
+
+import { CaseStatuses } from '../../../../../case/common/api';
+import { StatusFilter } from './status_filter';
+
+const stats = {
+  [CaseStatuses.open]: 2,
+  [CaseStatuses['in-progress']]: 5,
+  [CaseStatuses.closed]: 7,
+};
+
+describe('StatusFilter', () => {
+  const onStatusChanged = jest.fn();
+
+  it('should render', () => {
+    const wrapper = mount(
+      <StatusFilter
+        onStatusChanged={onStatusChanged}
```

Comment:
nit: these props are all the same. can you please put them in a `defaultProps` variable and spread them so it is easy to tell that